docs/outputs.rst

```diff
@@ -318,10 +318,13 @@ high-motion outliers.
 This file includes the high-motion volumes that are removed in most other derivatives.
 
 ``design.tsv`` is a tab-delimited file with one column for each nuisance regressor,
-including an intercept column, a linear trend column, and one-hot encoded regressors indicating
-each of the high-motion outlier volumes.
+including one-hot encoded regressors indicating each of the high-motion outlier volumes.
 This file includes the high-motion volumes that are removed in most other derivatives.
 
+.. important::
+   Please note that the outlier columns are somewhat misleading,
+   as volumes are removed by censoring, rather than regression.
+
```

Comment:
Is this also true if despiking is turned on?

Reply:
Despiking isn't represented in the design matrix, because it's done independently for each voxel and is only applied to the data (not the confounds).